src/Management/src/EndpointBase/SpringBootAdminClient/SpringBootAdminClientHostedService.cs

```diff
@@ -50,14 +50,25 @@ public async Task StartAsync(CancellationToken cancellationToken)
         app.Metadata.Merge(_options.Metadata);
 
         _httpClient.Timeout = TimeSpan.FromMilliseconds(_options.ConnectionTimeoutMS);
-        var result = await _httpClient.PostAsJsonAsync($"{_options.Url}/instances", app);
-        if (result.IsSuccessStatusCode)
+
+        HttpResponseMessage result = null;
+        try
+        {
+            result = await _httpClient.PostAsJsonAsync($"{_options.Url}/instances", app, cancellationToken: cancellationToken);
+        }
+        catch (Exception exception)
```

Comment:
Might be worth adding a comment that the generic `Exception` is being caught because ____ reason (I think the reasoning was that it does not matter what the exception is and we should log and continue because this is a non-essential operation) as this will show up at some point with Sonar rules

Reply:
I cannot imagine following that rule consistently across the code base. The better thing is to probably fix the sonar rules.